packages/core/lib/commands/develop/run.js

```diff
@@ -27,9 +28,17 @@ module.exports = async options => {
   const config = Config.detect(options);
   const customConfig = config.networks.develop || {};
 
-  const { mnemonic, accounts, privateKeys } = mnemonicInfo.getAccountsInfo(
-    customConfig.accounts || 10
-  );
+  const getAccounts = customConfig => {
+    if ("accounts" in customConfig) {
+      return mnemonicInfo.getAccountsInfo(customConfig.accounts);
```

Comment:
Ugh, this will cause a problem if the user supplies `null` or `undefined` for that value in the config. Should we first provide a guard here to check that `accounts` or `total_accounts` is a number or string representation of a number?

Reply:
Should we throw an error to the user that the supplied value should be a number?

Reply:
I'm not sure how much this is actually something to worry about?  Like, the previous method used in `develop`, where the code did `{...defaults, ...userOptions}` (not the exact code but YKWIM) also has this problem.

I would say the ideal behavior for `undefined` would be the same as if it isn't there, but I don't know if it's a big deal.  Whereas `null` strikes me as wholly not worth worrying about -- yeah if you do that you'll get problems, but why would you do that?

Reply:
I was also wondering why user will specify `null` as we didn't have any check for `null` previously too in our code. @eggplantzzz , what do you think about skipping the `null` check?